### DISCUSSION TITLE
How to monitor SC4S?

### DISCUSSION THREAD

**Discussion by twellinghurst** [splunk/splunk-connect-for-syslog#2303] [Q&A] [answered]
We recently implemented SC4S and had an issue where SC4S stopped sending specific syslog messages to Splunk, everything else was working as expected & there were no performance issues with the SC4S server. I restarted SC4S and the issue was resolved. Which prompts the question - how are others monitoring their SC4S instances to ensure it is functioning as expected?

**Comment by rjha-splunk** [comment]
we have a dashboard available to monitor SC4S.

**Comment by rjha-splunk** [comment, marked as answer]
https://splunk.github.io/splunk-connect-for-syslog/main/dashboard/
